src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+// component file
+import TodoContainer from './functionBased/components/TodoContainer';
+
+// stylesheet
+import './functionBased/App.css';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <Router>
+      <TodoContainer />
+    </Router>
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
```

Comment:
Great 🥳in using and importing the dependencies and you did well, but when I opened the project in my browser I found some errors in the console debugging💯, so it will be better if you fix the error⚠️Hint: the error says that it is no longer to use the render like that because it is not supported in the react 18, so kindly endeavor to fix this it will be better for you if the project has no errors for performance👌

To fix the error just follow [this documentation](https://bobbyhadz.com/blog/react-reacdom-render-no-longer-supported-in-react-18#:~:text=Use%20createRoot%20instead%22%20occurs%20because,element%20and%20use%20the%20ReactDOMClient.)⤵️

Reply:
Thank you for your feedback. I fixed the issue 👍